scripts/linters/pylint_extensions.py

```diff
@@ -1673,6 +1673,16 @@ class SingleLineCommentChecker(checkers.BaseChecker):
             'Please use a capital letter at the beginning of comment.',
             'no-capital-letter-at-beginning',
             'Please use capital letter to begin the content of comment.'
+        ),
+        'C0019': (
+            'Please use a single space at the beginning of pylint pragmas.',
+            'no-space-before-pylint-pragma',
+            'Please use a single space at the beginning of pylint pragmas.'
```

Comment:
This msg corresponds to the helper msg for this lint check, so it should describe what check it enforces - you can see other checkers as an example.